### DISCUSSION TITLE
Create GUI for easy initialisation of a new Wasp project

### DISCUSSION THREAD

**Discussion by matijaSos** [wasp-lang/wasp#971] [Archived]
@breadchris planted the seed for this one :) It could look something like this: https://start.spring.io/

We could maybe break it down further like this:
* v1 - very similar to the link above, users can specify whether they want js/ts, auth or not (and select providers) and other simple things
* v2 - we support more stuff from Wasp DSL - e.g. users can specify some initial routes, pages, and even draft data models (e.g. using some visual builder for Prisma/Postgres)
* v3 - we make this "real-time", so not just during initialisation - the user can always switch to GUI mode and see/change stuff there, in any moment during development

Huh this sounds like a really fun side project! :D

**Comment by sodic** [comment]
Related: https://github.com/wasp-lang/wasp/issues/899
